services/monolith/src/AuctionContext/Entities/Auction.ts

```diff
@@ -23,9 +27,24 @@ export class Auction {
     @Property({ type: 'string' })
     public readonly seller: string;
 
-    @Property({ type: TimeType })
+    @OneToMany({
+        entity: () => Bid,
+        mappedBy: 'auction',
+        orphanRemoval: true,
+        strategy: LoadStrategy.JOINED,
+        orderBy: { createdAt: QueryOrder.DESC },
+    })
+    public readonly bids = new Collection<Bid>(this);
+
+    @Property({ type: 'Date' })
     public readonly createdAt: Date;
 
+    @Property({ version: true })
+    public version: number = 1;
+
+    @Property({ type: 'number' })
+    public version2: number = 1;
```

Comment:
Used to make the ORM believe the entity is modified, remove before merge

Todo: Figure out how to force MikroORM to mark the entity as dirty

Reply:
Temporary solution: Added an updatedAt field.
Long term solution: Needs to be implemented in MikroORM